test/Sentinel1MetadataTest.jl

```diff
@@ -0,0 +1,236 @@
+"""
+unitests for the sentinel-1 metadata
+
+    test
+        1)MetaDataSentinel1
+        2)loading data 
+        3)the different sub structures.
+    
+"""
+
+include("../src/separateLater/Sentinel1/Metadata/exampleMetadata.jl")
+include("../src/separateLater/Sentinel1/Metadata/Sentinel1Metadata.jl")
+
+using Test
+import Dates
+
+
+#############################################
+########### Getting the .xml file ###########
+#############################################
+
+
+#dataPath = "data"
+# finding first .SAFE product... using that for testing.
+#safeFolder = (joinpath(dataPath, searchDir(dataPath, "SAFE")[1]))
+#getting annotation files in path
+#xmlFiles = getAnnotationPaths(safeFolder) #return vector of annotation files
+# taking first annotation file XML. 
+#const xmlFile = xmlFiles[1]
+
+const xmlFile = "test/testData/s1a-iw3-slc-vv-20220918t074921-20220918t074946-045056-056232-006.xml"
```

Comment:
This file should be included in runtests.jl.  Note that the relative xlm path might change